CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Changes:
 
 Bugfixes:
 - When using a Swift source, pseudo-directories are now recognized and transferred correctly.
+- Pass RequestOptions (headers like `X-Delete-At`) to segments.
```

Comment:
Since this description is user-facing, it should avoid developer-centric terms like `RequestOptions` and focus on the actual use case rather than on how it's implemented. For example:
> When uploading a segmented object to the target, expiration dates are now also applied to the segments. If you used an older version of swift-http-import to transfer files with expiration dates using segmented uploading, you will have to clean up those segments manually once the objects themselves have expired.